core/templates/components/summary-tile/exploration-summary-tile.component.css

```diff
@@ -0,0 +1,63 @@
+/*
+  NOTE to developers: We generate RTL CSS from this file in the webpack
+  compilation, here are some additional rules that can be added to the CSS files:
+  https://rtlcss.com/learn/usage-guide/control-directives .
+*/
+.oppia-exploration-dashboard-card {
```

Comment:
Please make sure that all rules are correctly scoped so that they just apply to this component. See #15724

Reply:
Ditto below.

Reply:
Done